cost/google/cud_recommendations/google_committed_use_discount_recommendations.pt

```diff
@@ -23,6 +23,23 @@ parameter "param_email" do
   description "Email addresses of the recipients you wish to notify when new incidents are created"
 end
 
+parameter "param_term" do
```

Comment:
Seeing as we have added two new parameters, do not forget to reflect this in the README for this policy template too

Reply:
@nia-vf1 , thank you Nia, I addressed this.